packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -0,0 +1,24 @@
+import { AxiosInstance } from 'axios';
+import { ISubscriberPayload } from './subscriber.interface';
+
+export class Subscribers {
+  private readonly http: AxiosInstance;
+
+  constructor(http: AxiosInstance) {
+    this.http = http;
+  }
+
+  async identify(subscriberId: string, data: ISubscriberPayload) {
+    return await this.http.post(`/subscribers`, {
+      subscriberId,
+      ...data,
+    });
+  }
+
+  async update(subscriberId: string, data: ISubscriberPayload) {
+    return await this.http.put(`/subscribers`, {
```

Comment:
```suggestion
    return await this.http.put(`/subscribers/${subscriberId}`, {
```

I think it should be like this in terms of valid REST protocol, or am I missing something?